docs/FAQ.md

```diff
@@ -68,13 +68,13 @@ Be careful as this is a destructive operation.
 Which is usually: `~/Library/Caches/Homebrew`
 
 ## My Mac `.app`s don’t find `/usr/local/bin` utilities!
-GUI apps on macOS don’t have `/usr/local/bin` in their `PATH` by default.
-If you’re on Mountain Lion, you can fix this by running
-`launchctl setenv PATH "/usr/local/bin:$PATH"`. [More details
-here](https://stackoverflow.com/questions/135688/setting-environment-variables-in-os-x/5444960#5444960),
-including how to set this across reboots. If you’re pre-Mountain Lion,
-[here’s an official
-alternative](https://developer.apple.com/legacy/library/qa/qa1067/_index.html).
+GUI apps on macOS don’t have `/usr/local/bin` in their `PATH` by
+default.  If you're on Mountain Lion or later, you can fix this by
```

Comment:
Pointing this out mostly for clarification from other maintainers: this PR changes a curly apostrophe to a straight one. Do we have a preference on which we use in docs? A naive `grep` through the source looks like a mix.

```
$ grep -r "'" * | wc -l
     235
[/usr/local/Homebrew/docs on ⇄ master]
$ grep -r "’" * | wc -l
     111
```

FWIW, I prefer the curlys because they look nicer, though they're certainly harder to type. (E.g. you may not I'm not using them in this comment.)